Templates/Java/requests_extensions/BaseEntityCollectionWithReferencesRequest.java.tt

```diff
@@ -45,44 +45,44 @@ import <#=importNamespace#>.concurrency.IExecutors;
     }
 
 <# if (c.GetFeatures().CanExpand) { #>
-    public <#=c.ITypeCollectionRequest()#> expand(final String value) {
+    public <#=c.ITypeCollectionWithReferencesRequest()#> expand(final String value) {
         addQueryOption(new com.microsoft.graph.options.QueryOption("$expand", value));
-        return (<#=c.TypeCollectionRequest()#>)this;
+        return this;
```

Comment:
Q: Is this getting removed because it was no-op or is there another reason?

Reply:
I removed it because it looked like an error and I don't think it ever worked to begin with (bad copy/paste)

TypeCollectionRequest will give us something like "GroupCollectionRequest" but `this` is a "GroupCollectionWithReferencesRequest". Even though they share some of the same methods, their first common ancestor is "BaseCollectionRequest".
On top of that the return type of the method is "ITypeCollectionWithReferencesRequest" which would give us "IGroupCollectionWithReferencesRequest", which is what the current class is implementing

Reply:
makes sense.